home/index.js

```diff
@@ -50,26 +52,43 @@ class MenteesView {
         `)
     }
 
-    static render(mentees) {
-        mentees.forEach(mentee => {
-            MenteesView.renderMentee(mentee);
+    static render(members) {
+        members.forEach(member => {
+            MembersView.renderMembers(member);
         });
     }
 }
 
 function run() {
-    FireBaseRequest.getMentees()
-        .then(mentees => {
-            MenteesView.render(mentees);
 
-            let modal = document.querySelector('.modal')
-            let infoButtons = document.getElementsByClassName('infoButton')
-            for (let i = 0; i < infoButtons.length; i++) {
-                infoButtons[i].addEventListener('click', function showModal() {
-                    const tagList = mentees[i].tags.map(tag => `
+    const value = localStorage.getItem("mentee");
+
+
+    let valClick;
+
+    // If user click mentee gets data from mentees collection, for mentors gets from mentors db collection.
+    if (value == 'true') {
+        let docRef = db.collection("mentees");
+        valClick = FireBaseRequest.getMembers(docRef)
+    } else {
+        let docRef = db.collection("mentors");
+        valClick = FireBaseRequest.getMembers(docRef)
+    }
+
+
+
+    // This shows rendered members in the cards.
+    valClick.then(members => {
```

Comment:
change docRef to memberRef and move
FireBaseRequest.getMembers(docRef) from lines 72 and 75
to line 81 before the "then" method